tests/unit/test_discovery_client.py

```diff
@@ -76,7 +77,7 @@ def test___discovery_service_available___unregister_non_registered_service___unr
 ):
     unregistration_success_flag = discovery_client.unregister_service()
 
-    assert ~unregistration_success_flag  # False
+    assert not unregistration_success_flag
 
 
 def test___discovery_service_unavailable___register_service_registration_failure(
```

Comment:
This test name is wrong and the code is now identical to `test___discovery_service_available___unregister_registered_service___unregistration_success`. 

I think your new tests cover the case where the discovery service is unavailable, so delete this test case.

Reply:
Deleted